introduction.md

```diff
@@ -109,36 +105,23 @@ As a consequence of the N-1 resilience model, there are multiple attacks against
 
 
-### Underlying Assumptions
+### Transaction Fee Risk
 
-#### Bitcoin
+A fundamental issue arises with pre-signed transactions commiting to the feerate well in advance of the time of broadcast. The required feerate is unpredictable and a transaction with a low feerate may not be included in a block in time.
 
-- assume users can propagate revocation transactions to miners.
-- assume miners will mine the transaction paying the most feerate, regardless of any other (meta)data.
-- assume a significant hashrate to prevent reorgs of depth higher than the unvault time-lock length.
+[Pinning attacks](https://bitcoinops.org/en/topics/transaction-pinning/) are possible when using either RBF or CPFP to dynamically allocate fees to transactions. These attacks exploit vulnerabilities that can cause transactions to become "stuck" in the mempool for long times. For protocols like Revault which require timely transaction confirmation, these attacks can be used to defeat theft-mitigation features (e.g. pinning a Cancel TX for longer than the Unvault TX's time-lock, and stealing funds).
 
-#### Infrastructure
+The Unvault TXs have an additional output that allows dynamically allocating fees through child-pays-for-parent (CPFP). This feature helps managers optimise withdrawal times. The possibility of pinning Unvault TXs doesn't impact the theft-mitigation features of Revault but can be used as a denial-of-service attack by comrpomised managers. This is acceptable since managers already have multiple ways to launch denial-of-service attacks. For the Cancel, Emergency and Unvault Emergency TXs it is critical to avoid transaction pinning attacks. Unfortunately, these transactions aren't compatible with Bitcoin's [CPFP carve-out](https://github.com/bitcoin/bitcoin/pull/15681) rule which is used to mitigate pinning attacks in Lightning.
 
-- assume a proper ceremony (FIXME: link) as the root of trust for cryptographic protocols used.
-- assume a secure signing device allowing users to check the validity of the data being signed.
-- assume at least one honest cosigning server AND at least one honest online watchtower per stakeholder.
-- assume the N stakeholders’ keys to not all be compromised.
-- assume sufficient fees for Revault TXs such that operations aren't stalled. 
-- assume the Unvault TX output relative timelock to be high enough for an attack on the block space market to not be worth it.
\ No newline at end of file
+Emergency and Unvault Emergency TXs are simply pre-signed with exorbitant fees as they are intended as a deterrent, and not a regular expense. Since Cancel TXs may (or may not) occur frequently, it is important to minimise their feerates while still ensuring timely confirmation. The best solution currently is to prepare multiple variants of the same Cancel TX with a broad distribution of feerates, from typical to extreme and with many in-between. At the time of broadcast, the one with the appropriate fee should be selected.
```

Comment:
Meh, it's not what the specs are saying. Let's keep that to when we modify the spec to not use ACP anymore. Maybe even the previous paragraph.

Reply:
Ok I removed it for now